packages/components/select/src/useSelect.ts

```diff
@@ -225,6 +225,14 @@ export const useSelect = (props, states: States, ctx) => {
     () => states.visible,
     (val) => {
       if (!val) {
+        if (props.filterable) {
+          if (typeof props.filterMethod === 'function') {
```

Comment:
```suggestion
          if (isFunction(props.filterMethod)) {
```

Maybe use the `isFunction` functions from @element-plus/utils

Reply:
`typeof ...'function'` also used in else where in this file, so had better optimise it all by others?

Reply:
I think you only need to modify your code that fixes the bug you describe. The others can be refactored in a new PR.

Reply:
> I think you only need to modify your code that fixes the bug you describe. The others can be refactored in a new PR.

yes i fixed it, so can i close it?

Reply:
I didn't see it. Did you push the code to github?

Reply:
> I didn't see it. Did you push the code to github?
Have  you seen the commit code `36c2286`, it's my commit.

Reply:
> I didn't see it. Did you push the code to github?

![图片](https://user-images.githubusercontent.com/75945382/190205486-7bba242c-c56b-4896-ba41-4b5b6ec21506.png)
the mark one is my code and the next one is a extra commit.

Reply:
> `typeof ...'function'` also used in else where in this file, so had better optimise it all by others?

Other files do not need to be considered. but for your changes, you should write in the format we recommend.

Reply:
> > `typeof ...'function'` also used in else where in this file, so had better optimise it all by others?
> 
> Other files do not need to be considered. but for your changes, you should write in the format we recommend.

I've already made changes in the latest commit, please check to see.